test/e2e/tinkerbell_test.go

```diff
@@ -1596,6 +1596,17 @@ func TestTinkerbellKubernetes127UbuntuSimpleFlow(t *testing.T) {
 	runTinkerbellSimpleFlow(test)
 }
 
+func TestTinkerbellKubernetes128UbuntuSimpleFlow(t *testing.T) {
```

Comment:
Do we need a test for Ubuntu 2204 as well? Or is that not supported for Tinkerbell as yet?

Reply:
yeah, thanks for catching that